### ISSUE TITLE
kibana 6.1 plugin

### ISSUE THREAD

**Issue by theburakcinar** [wazuh/wazuh-kibana-app#132]
hi , 

kibana-6.1.0-1.x86_64 installed on centos 7 machine.  when i try to install wazuh plugin it says "Incorrect Kibana version in plugin [wazuh]. Expected [6.1.0]; found [6.0.1]" .

could you check ?

/usr/share/kibana/bin/kibana-plugin install https://packages.wazuh.com/wazuhapp/wazuhapp.zip
Attempting to transfer from https://packages.wazuh.com/wazuhapp/wazuhapp.zip
Transferring 4577607 bytes....................
Transfer complete
Retrieving metadata from plugin archive
Extracting plugin archive
Extraction complete
Plugin installation was unsuccessful due to error "Incorrect Kibana version in plugin [wazuh]. Expected [6.1.0]; found [6.0.1]"

**Comment by jesusgn90** [comment]
Hi @theburakcinar sorry about this issue but Elastic just released a 6.1.0 version of Kibana that has some breaking changes and our plugin is directly affected. We are working to have a new App version in order to work properly with that Kibana version.

You can still fix your trouble using the **6.0.1** version of Kibana, so assuming you have installed ELK 6.1.0 stack you must to **remove all of them** and try to install again targeting the packages with version 6.0.1 of the ELK.

CentOS like OS:
```
# yum remove kibana
# yum remove logstash
# yum remove elasticsearch
```

Debian/Ubuntu
```
# apt-get remove kibana
# apt-get remove logstash
# apt-get remove elasticsearch
```
Then **follow again the documentation**, but when install the ELK packages add **6.0.1**, so you should install them as follow:

CentOS like OS:
```
# yum install kibana-6.0.1
# yum install logstash-6.0.1
# yum install elasticsearch-6.0.1
```

Debian/Ubuntu
```
# apt-get install kibana=6.0.1
# apt-get install logstash=6.0.1
# apt-get install elasticsearch=6.0.1
```

**Comment by jesusgn90** [comment]
__Update__:

Since we have just **released** a Wazuh App package that is working the Elastic Stack **6.1.0** version, now you are able to make it work as usual with the installation guide. You can stay at 6.0.1 stack but now, you need to use this Wazuh App package: https://packages.wazuh.com/wazuhapp/wazuhapp-3.0.0_6.0.1.zip, If you still have troubles please let me know and please **open** a new issue,  I understand that this issue is solved.  

Best regards,
Jesús.
